src/shapes/Text.ts

```diff
@@ -682,6 +697,22 @@ Factory.overWriteSetter(Text, 'width', getNumberOrAutoValidator());
 
 Factory.overWriteSetter(Text, 'height', getNumberOrAutoValidator());
 
+
+/**
+ * get/set direction
+ * @name Konva.Text#direction
+ * @method
+ * @param {String} direction
+ * @returns {String}
+ * @example
+ * // get direction
+ * var direction = text.direction();
+ *
+ * // set direction
+ * text.direction('rtl');
+ */
+Factory.addGetterSetter(Text, 'direction', 'ltr');
```

Comment:
Should it be `inherit` by default?

Reply:
For now, I think inherit will work just fine.
There may be a global page option for direction. So I think Konva should preserve it when possible. 

To detect what direction to apply, we can probably use this:
```js
const textDir = this.direction();
const canvasDir = canvas.direction;
const direction = textDir === 'inherit' ? canvasDir : textDir;
// then do rest of the logic in sceneFunc
```